packages/tools/devtools/devtools-browser-extension/README.md

```diff
@@ -111,6 +111,11 @@ To use a local build of this extension in your browser:
 <!-- prettier-ignore-start -->
 <!-- NOTE: This section is automatically generated using @fluid-tools/markdown-magic. Do not update these generated contents directly. -->
 
+## Inject telemetry key
+If you're manually releasing or testing the Devtools browser extension using a local build, create your .env file with content like  "DEVTOOLS_TELEMETRY_TOKEN=abcdefgh-ijkl-mnop-qrst-uvwxyz6ffd9c". In automated pipeline builds, the telemetry key will be fetched from the "prague-key-vault" group in Azure Ops, and output the generated extension in artifact named `devtools-extension-bundle`.
```

Comment:
Do devs need to manually create this file and add the key? Or does `npm start` create the file / contents automatically?

Reply:
I think it should be always manual when done locally. Unless we need a valid token locally for some reason (testing, most probably), the `.env` file doesn't need to exist (I'd argue maybe even slightly better that it doesn't exist) so I wouldn't make the npm scripts create one.